testutil/testdata/test_tx.go

```diff
@@ -54,7 +54,11 @@ func (msg *TestMsg) GetSignBytes() []byte {
 func (msg *TestMsg) GetSigners() []sdk.AccAddress {
 	addrs := make([]sdk.AccAddress, len(msg.Signers))
 	for i, in := range msg.Signers {
-		addr, _ := sdk.AccAddressFromBech32(in)
+		addr, err := sdk.AccAddressFromBech32(in)
+		if err != nil {
+			panic(err)
+		}
+
```

Comment:
this function is used only in tests. How about passing `t *testing.T` as an argument, and instead of `panic` we can do `t.Fatal(...)`

Reply:
if it's not meant to be used in tests, then we shouldn't `panic`.

Reply:
Why? Tests were failing because this part of the code was returning an error that was subsequently ignored. Considering that `GetSigners()` is not meant to return an eror, if at this stage there's something wrong, then this piece of code must crash and make as much noise as possible - and AFAIU that's what `panic()` is for.

However, as I said above, I'm not going to take care of it in this PR.